cpp_exercises/day2/date/tests/tst_date.cpp

```diff
@@ -21,6 +21,10 @@ private slots:
     void equalityOperatorTest();
     void addYearsTest();
     void addMonthsTest();
+    /*!
+     * results of the tests taken from https://www.timeanddate.com/date/dateadd.html
+     */
```

Comment:
Anche questo commento lo lascerei in corrispondenza dell'implementazione del test, altrimenti una persona che arriva per aggiungere qualche test-case non si accorge di questo commento in cima.